mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -271,6 +260,20 @@ def _rpc(self, method, *args):
             raise ProtocolError(ProtocolError.MISMATCHED_API_ID)
         return result['result']
 
+    def _is_app_running(self):
```

Comment:
Sure we can fork it at some point, but you mentioned this specific bug was one you'd seen in sl4a too right?

Also sl4a isn't the only thing; snippetlib 1.0.0 is out now. This change would make us incompatible with 1.0.0 which people might be using. So we have to make protocol-breaking changes very carefully and with lots of notice to users. :)

In this case I think it's easy enough to work around by reusing one of the existing RPCs that it's not worth trying to change the protocol. Connect was just the easiest way but we could implement it as some kind of erroroneous RPC if you want.